platform/cxxsupport/mstd_cstddef

```diff
@@ -26,9 +26,14 @@
  * - For all toolchains:
  * - - MSTD_CONSTEXPR_XX_14 macros that can be used to mark
  *     things that are valid as constexpr only for C++14 or later,
+ *     permitting constexpr use where ARM C 5 would reject it.
+  * - - MSTD_CONSTEXPR_XX_11 macros that can be used to mark
+ *     things that are valid as constexpr only for C++11 or later,
```

Comment:
The `_11` macros are actually needed to permit `constexpr` alternatives for C, rather than anything to do with ARM C 5